WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -167,17 +173,20 @@ public class CurrencyFormatter {
     ///  - 5800199.56 becomes "$5.8m"
     ///
     func formatHumanReadableAmount(_ stringAmount: String,
-                                   with currency: String = CurrencySettings.shared.currencyCode.rawValue,
-                                   roundSmallNumbers: Bool = true) -> String? {
-        guard let amount = convertToDecimal(from: stringAmount) else {
+                                   with currency: String? = nil,
+                                   roundSmallNumbers: Bool = true,
+                                   locale: Locale = .current) -> String? {
+        guard let amount = convertToDecimal(from: stringAmount, locale: locale) else {
             return nil
```

Comment:
Just a small improvement since we are touching this class: maybe we can add an `assertionFailure` here? It can be helpful for debugging purposes.